Formula/dotnet.rb

```diff
@@ -0,0 +1,88 @@
+class Dotnet < Formula
+  desc ".Net Core"
+  homepage "https://github.com/dotnet/cli/releases"
+  url "https://github.com/dotnet/cli/archive/v1.0.0-preview2.tar.gz"
```

Comment:
Still better to use a GitHub checkout here and it'll remove the need for the patch.